src/main/java/core/basesyntax/Flatness.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Flatness {
+
+    public abstract double getArea();
```

Comment:
```suggestion
      double getArea();
```
they are implicitly public and abstract